src/site/pages/Pair/index.jsx

```diff
@@ -0,0 +1,133 @@
+import { pick } from 'lodash'
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { withRouteData } from 'react-static'
+import { createStructuredSelector } from 'reselect'
+import { compose, setDisplayName, lifecycle, withProps, withState } from 'recompose'
+import { Container, Row, Col } from 'reactstrap'
+
+import PriceChart from 'Components/PriceChart'
+
+// import logoImg from 'Img/faast-logo.png'
+import Header from 'Site/components/Header'
+import Footer from 'Site/components/Footer'
+import SwapWidget from 'Site/components/SwapWidget'
+import Features from 'Site/components/Features'
+
+import { fetchGeoRestrictions } from 'Common/actions/app'
+import { retrieveAssets } from 'Common/actions/asset'
+import { getAllAssetsArray, areAssetsLoaded } from 'Common/selectors/asset'
+
+import MoonBackground from 'Img/moon-background.jpg'
+import MacbookScreenshot1 from 'Img/macbook-screenshot-01.png'
+
+import classNames from 'class-names'
+
+import { selectedChartButton } from './style.scss'
+
+export default compose(
+  setDisplayName('Pairs'),
+  connect(createStructuredSelector({
+    assets: getAllAssetsArray,
+    areAssetsLoaded: areAssetsLoaded,
+  }), {
+    retrieveAssets,
+    fetchGeoRestrictions
+  }),
+  withProps(({ assets }) => ({
+    assetList: assets.filter(({ deposit, receive }) => deposit || receive)
+      .map((asset) => pick(asset, 'symbol', 'name', 'iconUrl'))
+  })),
+  withState('selectedChart', 'updateSelectedChart', undefined),
+  lifecycle({
+    componentWillMount() {
+      const { fetchGeoRestrictions, retrieveAssets } = this.props
+      fetchGeoRestrictions()
+      retrieveAssets()
+    }
+  }),
+  withRouteData,
+)(({ supportedAssets, areAssetsLoaded, assetList, pair, updateSelectedChart, selectedChart, 
+  descriptions }) => {
+  supportedAssets = areAssetsLoaded ? assetList : supportedAssets
+  const defaultDeposit = pair.split('-')[0]
+  const defaultReceive = pair.split('-')[1]
+  selectedChart = selectedChart ? selectedChart : defaultDeposit
+  return (
+    <div>
+      <div>
+        <Header/>
+        <div className='jumbotron jumbotron-fluid hero-technology mb-0' style={{
```

Comment:
It looks like this jumbotron is duplicated in the Home component with the only a couple of differences. Consider refactoring it into its own component so that we only have to change text and image content in one place down the road.